api/all.yaml

```diff
@@ -2145,6 +2203,89 @@ components:
         - PATH_CREATE_TABLE
         - PATH_REFRESH
       type: string
+    PrincipalType:
+      type: string
+      enum:
+        - USER
+        - GROUP
+      description: The type of the principal.
+    SecurableType:
+      type: string
+      enum:
+        - metastore
+        - catalog
+        - schema
+        - table
+        - function
+        - volume
+        - registered-model
```

Comment:
@kriscon-db is there a `registered-model` securable type or are models covered under the `function` securable type? 

Reference: https://docs.databricks.com/api/workspace/grants/update#securable_type

Reply:
> https://docs.databricks.com/api/workspace/permissions/get

This is workspace permissions endpoint, not the catalog permissions endpoint.

Reply:
It implements the UC endpoints. The pointer was to show it may just be missing from the API docs. @kriscon-db can verify. it may be that it _should be_ `registered_model` though. I can make that change if that is correct.